src/preludeTemplate.js

```diff
@@ -152,15 +152,15 @@
             prepareRealmGlobalFromConfig(moduleRealm.global, globalsConfig, endowments, globalStore)
             // execute in module realm with modified realm global
             try {
-              moduleInitializer = moduleRealm.evaluate(`${moduleSource}`)
+              moduleInitializer = moduleRealm.evaluate(moduleSource)
```

Comment:
This idiom can be used to ensure that the result is a string. It's not always a noop.